integration-tests/src/test/java/org/apache/fineract/integrationtests/common/GlobalConfigurationHelper.java

```diff
@@ -99,8 +99,8 @@ public static void verifyAllDefaultGlobalConfigurations(final RequestSpecificati
         ArrayList<HashMap> actualGlobalConfigurations = getAllGlobalConfigurations(requestSpec, responseSpec);
 
         // There are currently 37 global configurations.
```

Comment:
Delete the comment please or at least update it. Worst comment is the misleading comment.

Reply:
done